### PR TITLE
fix: notifications page inaccuracies

### PR DESCRIPTION
**Pull request by sshanzel**
### Describe what this PR does
- [x] notification permission banner is missing its BG
- [x] avatars are bigger than expected and the tooltip doesn’t work
- [x] the notification item is supposed to be a link to be middle clickable
- [x] border between notification items

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

WT-{number} #done

### CODE REVIEW

File: packages/shared/src/components/notifications/NotificationItem.tsx [hunk 1]
```diff
@@ -51,47 +52,45 @@ function NotificationItem({
...
-        isUnread && 'bg-theme-float',
-      )}
-    >
-      <NotificationItemIcon icon={icon} />
-      <div className="flex flex-col flex-1 ml-4 w-full text-left typo-callout">
-        {hasAvatar && (
-          <span className="flex flex-row gap-2 mb-4">{avatarComponents}</span>
+    <Link href={targetUrl} passHref>
+      <a
+        href="#"
```
Comment (idoshamun) [inline packages/shared/src/components/notifications/NotificationItem.tsx:58]: I think this is a mistake right?

File: packages/shared/src/components/notifications/NotificationItemAvatar.tsx [hunk 2]
```diff
@@ -18,7 +20,16 @@ function NotificationItemAvatar({
 
   if (type === NotificationAvatarType.User) {
     return (
-      <ProfilePicture user={{ image, id: referenceId }} nativeLazyLoading />
+      <ProfileTooltip
+        link={{ href: `${webappUrl}/${referenceId}` }}
```
Comment (idoshamun) [inline packages/shared/src/components/notifications/NotificationItemAvatar.tsx:24]: Avatar has the property `targetUrl` please use it instead

**Review by idoshamun — CHANGES_REQUESTED** [review]
Few changes

File: packages/shared/src/components/notifications/FirstNotification.tsx [hunk 3]
```diff
@@ -20,6 +20,7 @@ function FirstNotification(): ReactElement {
       isUnread={isUnread}
       type={NotificationType.System}
       icon={NotificationIcon.Bell}
+      targetUrl="#"
```
Comment (el) [inline packages/shared/src/components/notifications/FirstNotification.tsx:23]: I dont think you should do that

File: packages/shared/src/components/notifications/NotificationItem.tsx [hunk 4]
```diff
@@ -51,47 +52,45 @@ function NotificationItem({
...
-        'flex flex-row py-4 pl-6 pr-4 hover:bg-theme-hover focus:bg-theme-active',
-        isUnread && 'bg-theme-float',
-      )}
-    >
-      <NotificationItemIcon icon={icon} />
-      <div className="flex flex-col flex-1 ml-4 w-full text-left typo-callout">
-        {hasAvatar && (
-          <span className="flex flex-row gap-2 mb-4">{avatarComponents}</span>
+    <Link href={targetUrl} passHref>
+      <a
```
Comment (el) [inline packages/shared/src/components/notifications/NotificationItem.tsx:57]: Please use `button` if it doesnt have a target URL, and use `a` if it does with the href

**Review by el — CHANGES_REQUESTED** [review]

File: packages/shared/src/components/notifications/FirstNotification.tsx (hunk 3 above)
Comment (sshanzel) [reply packages/shared/src/components/notifications/FirstNotification.tsx:23]: There is nothing to do here in the first notification. Not sure if there should be any action here.

File: packages/shared/src/components/notifications/NotificationItem.tsx (hunk 4 above)
Comment (sshanzel) [reply packages/shared/src/components/notifications/NotificationItem.tsx:57]: Every notification will have a `targetUrl`.

File: packages/shared/src/components/notifications/NotificationItem.tsx (hunk 1 above)
Comment (sshanzel) [reply packages/shared/src/components/notifications/NotificationItem.tsx:58]: We pass the href, from the `Link` component, so this gets overwritten. Removing it makes the linter go crazy.

File: packages/shared/src/components/notifications/NotificationItemAvatar.tsx (hunk 2 above)
Comment (sshanzel) [reply packages/shared/src/components/notifications/NotificationItemAvatar.tsx:24]: Oh, yep. That is much better. Let me change this.

File: packages/shared/src/components/notifications/FirstNotification.tsx (hunk 3 above)
Comment (sshanzel) [reply packages/shared/src/components/notifications/FirstNotification.tsx:23]: We still don't have the rebrandly link from the product, let's add it in the intermediate branch so we can close out this PR.

Comment (el) [reply packages/shared/src/components/notifications/FirstNotification.tsx:23]: Yeah, ok for me 👍

**Review by el — APPROVED** [review]

File: packages/shared/src/components/notifications/FirstNotification.tsx (hunk 3 above)
Comment (sshanzel) [reply packages/shared/src/components/notifications/FirstNotification.tsx:23]: Thank you for the review! 🚢

**Pull request merged**